homeassistant/components/teslemetry/entity.py

```diff
@@ -78,12 +77,7 @@ def has(self) -> bool:
 
     async def handle_command(self, command) -> dict[str, Any]:
         """Handle a command."""
-        try:
-            result = await command
-        except TeslaFleetError as e:
-            raise HomeAssistantError(f"Teslemetry command failed, {e.message}") from e
-        LOGGER.debug("Command result: %s", result)
-        return result
+        return await handle_command(command)
```

Comment:
This method can be removed, as it now is just a proxy

Reply:
Do you mean I should import and use it directly in each platform?

Reply:
Yeah? As in, this method doesn't do anything besides playing a proxy

Reply:
Ok done.